packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -269,41 +269,24 @@ class RenderAndroidView extends PlatformViewRenderBox {
   }
 }
 
-/// A render object for an iOS UIKit UIView.
-///
-/// [RenderUiKitView] is responsible for sizing and displaying an iOS
-/// [UIView](https://developer.apple.com/documentation/uikit/uiview).
-///
-/// UIViews are added as sub views of the FlutterView and are composited by Quartz.
-///
-/// {@macro flutter.rendering.RenderAndroidView.layout}
-///
-/// {@macro flutter.rendering.RenderAndroidView.gestures}
-///
-/// See also:
-///
-///  * [UiKitView] which is a widget that is used to show a UIView.
-///  * [PlatformViewsService] which is a service for controlling platform views.
-class RenderUiKitView extends RenderBox {
-  /// Creates a render object for an iOS UIView.
+/// Superclass for common fucntionality of platform view RenderBoxes
```

Comment:
Can you add a little more color to this doc? See also our style guide for writing prompts: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#writing-prompts-for-good-documentation

Questions to answer: 
* What is the common functionality that this provides?
* What responsibilities are left to the subclasses?
* Can this class be use directly? (it's documented as a super class, but it isn't abstract, so what do I get if I instantiate it directly?)
* Document what the generic type parameter `T` is

Reply:
> What responsibilities are left to the subclasses?

Looks like at least gesture handling needs to be provided by the sub class?